examples/playground/cypress/integration/table/ActionableRows_spec.js

```diff
@@ -0,0 +1,23 @@
+describe("Actionable Rows", function () {
+	beforeEach(function () {
+		cy.visit("http://localhost:21356/#/components/table");
+	});
+
+	context("Navigate to Actionable Rows Table", function () {
+		it("can navigate to 'actionable rows' from tab options", function () {
+			cy.get(".btn-group").within(function () {
+				cy.contains("actionable rows")
+					.click();
+			});
+			cy.get(".table-responsive").within(function () {
```

Comment:
@leayousoufov  Instead of checking the table data (which may change) I believe you can look at the active class of the button-group to find what page we're on.  Also, checking the Table Title would be a good idea.